File: Python/1015 - Distance Between Two Points.py
import math
s1 = input()
l1 = s1.split(" ")
s2 = input()
l2 = s2.split(" ")
A = pow((float(l2[0]) - float(l1[0])),2)
B = pow((float(l2[1]) - float(l1[1])),2)
DIS = math.sqrt(A+B)
print("%.4f" % DIS)

File: C++/1015 - Distance Between Two Points.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
 
int main() {
    double x1,y1,x2,y2,distance;
    std::cin >> x1 >> y1;
    std::cin >> x2 >> y2;
    distance = sqrt((pow(x2-x1,2)+pow(y2-y1,2)));
    printf("%.4f\n", distance);
    return 0;
}